src/renderer/mod.rs

```diff
@@ -307,9 +313,22 @@ impl Renderer {
         };
 
         let settings = self.settings.get::<RendererSettings>();
+        let size = root_canvas.base_layer_size();
+        let screen_rect = Rect::from_xywh(0.0, 0.0, size.width as f32, size.height as f32);
         let root_window_regions = root_windows
             .into_iter()
-            .map(|window| window.draw(root_canvas, default_background, grid_scale))
+            .map(|window| {
+                window.draw(
+                    root_canvas,
+                    (opacity * 255.0) as u8,
+                    Background {
+                        color: default_background.with_a((255.0 * background_transparency) as u8),
+                        image: background_image,
+                    },
+                    &screen_rect,
+                    grid_scale,
+                )
+            })
```

Comment:
There's something wrong here, probably with these changes, but it could be something else. Without setting any transparency option, or background image, the border becomes transparent

![image](https://github.com/user-attachments/assets/0e5aa650-59d6-41ff-9787-e9e1b46a23c9)